### PR TITLE
Adding RAC and AVFoundation extensions tests

### PR DESCRIPTION
**Pull request by FranDepascuali**
## Sumary ##
**Trello card**: https://trello.com/c/5ldD3wbp/10-agregar-tests

Adding RAC and AVFoundation tests

### CODE REVIEW

**Comment by woloxci** [comment]
:+1: There are no linter violations.

File: CoreTests/Extensions/AVFoundation/AVAssetSpec.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Nimble
+import Core
+
+// TODO: Implement AVAssetSpec
+public class AVAssetSpec: QuickSpec {
+    
+    override public func spec() {
+        
+        describe("#loadValuesAsynchronouslyForKeys") {
+
```
Comment (danielaRiesgo) [inline CoreTests/Extensions/AVFoundation/AVAssetSpec.swift]: Should here be tests? If it cannot be tested then eliminate the file.

File: CoreTests/Extensions/AVFoundation/AVPlayerItemSpec.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import Nimble
+import Core
+
+// TODO: Implement AVPlayerItemSpec
+public class AVPlayerItemSpec: QuickSpec {
+    
+    override public func spec() {
+        
+        describe("#seekTo") {
+
```
Comment (danielaRiesgo) [inline CoreTests/Extensions/AVFoundation/AVPlayerItemSpec.swift]: Should here be tests? If it cannot be tested then eliminate the file.

File: CoreTests/Extensions/ReactiveCocoa/SignalProducerSpec.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                
+                var observer: Observer<(), NSError>!
+                
+                beforeEach {
+                    let (_producer, _observer) = SignalProducer<(), NSError>.buffer(0)
+                    producer = _producer
+                    observer = _observer
+                }
+                
+                it("should ignore the error and complete") {
```
Comment (danielaRiesgo) [inline CoreTests/Extensions/ReactiveCocoa/SignalProducerSpec.swift]: Maybe we could send one value to show that it only ignores errors but not values.

File: CoreTests/Extensions/ReactiveCocoa/SignalSpec.swift [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                
+                var observer: Observer<(), NSError>!
+                
+                beforeEach {
+                    let (_signal, _observer) = Signal<(), NSError>.pipe()
+                    signal = _signal
+                    observer = _observer
+                }
+                
+                it("should ignore the error and complete") {
```
Comment (danielaRiesgo) [inline CoreTests/Extensions/ReactiveCocoa/SignalSpec.swift]: Maybe we could send one value to show that it only ignores errors but not values.

**Comment by danielaRiesgo** [comment]
👍

**Comment by woloxci** [comment]
:+1: There are no linter violations.

**Pull request merged**